### PR TITLE
Added missing- , sorted and re-arranged permissions

### PR DESCRIPTION
**Pull request by EliteArtz**
**Please describe the changes this PR makes and why it should be merged:**
Added missing- , sorted and re-arranged options/permissions
**Status**

- [x] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [x] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

### CODE REVIEW

**Review by almeidx — CHANGES_REQUESTED** [review]
Please do not change the `permission` field values as to not make backwards-incompatible changes. Note that the HTML tags can be arranged as you like, what matters is the `value` attribute

**Comment by EliteArtz** [comment]
> Please do not change the `permission` field values as to not make backwards-incompatible changes. Note that the HTML tags can be arranged as you like, what matters is the `value` attribute

I was aware of that, that's why I changed the cases in the switch to correspond to the right option.

**Comment by EliteArtz** [comment]
I will commit one more to adjust the rest.

**Comment by almeidx** [comment]
Adjusting the switch statement will not change the fact that it will be backwards-incompatible. Let's say, hypothetically, you were using this action and were setting the "Manage Guild" permission (value is "1"). After this change, as it is, the permission will now be changed to "View Channels"

**Comment by EliteArtz** [comment]
The way how it works is through the values set in the `permission` select field, the saved variable `info` gets the current selected value, which then corresponds to the switch, which is using this `info` variable to define which bigint permission is about to used.

It will be saved to `result` and then it does the api request to Discord via the `setPermissions` Promise function.

The mod changes has been tested by me with good results.

**Comment by TheMonDon** [comment]
That is not what alm meant, what alm is trying to say is you can't change the field value as it breaks existing commands and we try our best to not break it for backward comparability.

**Comment by EliteArtz** [comment]
I see now..
Alright I am rearranging the values. The next commit should be hopefully final.

File: actions/set_role_perms_MOD.js [hunk 1]
```diff
@@ -230,28 +266,59 @@ module.exports = {
...
+        break;
       default:
         break;
     }
 
     if (role?.id) {
       const perms = role.permissions;
-      if (Array.isArray(role)) {
+
+      if (Array.isArray(role))
```
Comment (almeidx) [inline actions/set_role_perms_MOD.js:311]: This will never happen due to the if statement before this one (arrays don't have an `id` property)

**Review by almeidx — COMMENTED** [review]
We should be using the [`Permissions.FLAGS`](https://discord.js.org/#/docs/discord.js/stable/class/Permissions?scrollTo=s-FLAGS) object instead of hardcoding the permission values, but its not a blocker

**Comment by EliteArtz** [comment]
> We should be using the [`Permissions.FLAGS`](https://discord.js.org/#/docs/discord.js/stable/class/Permissions?scrollTo=s-FLAGS) object instead of hardcoding the permission values, but its not a blocker

It would probably be better, yeah let's try to do that.
Either way, the Array if was not coded by me, but I'll be sure handle it.

**Review by almeidx — APPROVED** [review]

**Pull request merged**